nats-ballerina/src/nats/basic/producer.bal

```diff
@@ -81,22 +82,22 @@ public client class Producer {
     }
 }
 
-function producerInit(Connection c) =
+isolated function producerInit(Connection c) =
 @java:Method {
     'class: "org.ballerinalang.nats.basic.producer.Init"
 } external;
 
-function closeConnection(Producer producer) =
+isolated function closeConnection(Producer producer) =
 @java:Method {
     'class: "org.ballerinalang.nats.basic.producer.CloseConnection"
 } external;
 
-function externRequest(Producer producer, string subject, Content data, int? duration = ()) returns Message | Error =
-@java:Method {
+isolated function externRequest(Producer producer, string subject, Content data, int? duration = ())
+returns Message | Error = @java:Method {
     'class: "org.ballerinalang.nats.basic.producer.Request"
 } external;
 
-function externPublish(Producer producer, string subject, string | byte[] data, (string | service)? replyTo = ())
-returns Error? = @java:Method {
+isolated function externPublish(Producer producer, string subject, string | byte[] data, (string | service) ? replyTo
+= ()) returns Error? = @java:Method {
```

Comment:
It would be better to break the line as follows:
```suggestion
isolated function externPublish(Producer producer, string subject, string | byte[] data, 
(string | service) ? replyTo = ()) returns Error? = @java:Method {
```